src/dipdup/datasources/datasource.py

```diff
@@ -38,6 +40,26 @@ def set_logger(self, name: str) -> None:
         self._logger = FormattedLogger(self._logger.name, name + ': {}')
 
 
+class HttpDatasource(Datasource):
+    _default_http_config = HTTPConfig(
+        cache=True,
+        retry_count=5,
+        retry_sleep=1,
+        ratelimit_rate=0,
+        ratelimit_period=0,
+    )
+
+    def __init__(self, http_config: Optional[HTTPConfig] = None) -> None:
+        super().__init__('', self._default_http_config.merge(http_config))
+        self._logger = _logger
+
+    async def get(self, url: str, cache: bool = False, weight: int = 1, **kwargs):
```

Comment:
Nice idea. I think it's better to add helpers for all basic HTTP methods in `HttpGateway`. One may say it's not necessary to add a bunch of methods that hide a single argument, but this is expected behavior for `requests` and `aiohttp` users.

Reply:
The only reason to add this method here is that the request method of `HttpGateway` joins path from the `url` call parameter and domain from the `url` datasource settings, which is not provided for `HttpDatasource` by design.